src/AppBundle/Controller/Admin/AdminProjectController.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace AppBundle\Controller\Admin;
+
+use AppBundle\Entity\Project;
+use AppBundle\Form\AddProjectType;
+use AppBundle\Form\AdminLoginType;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ *
+ * @Route("/admin/projects")
+ */
+class AdminProjectController extends Controller
+{
+    /**
+     * @Route("/show", name="admin_projects_show")
+     * @Template()
+     * @Method({"GET"})
+     */
+    public function listAction()
+    {
+        $security = $this->get('security.context');
+        if ($security->isGranted('ROLE_ADMIN')) {
```

Comment:
It's already was wrote [this](https://github.com/civils-council/civilsbudget/blob/api/app/config/security.yml#L60)